tests/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'Omer'
```

Comment:
That allows running a specific commands such as `python -m unittest tests.test_bounding_box_query`
Convenient for testing locally, but I can remove it you don't want it on git